<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Overload</class>
 <widget class="QMainWindow" name="Overload">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_result">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>191</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cmp">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> 
QPushButton#pushButton_cmp {
    
  backface-visibility: hidden;
 
  cursor: pointer;
  
  
  background: #97311a;
  border-radius: 100px;
  border: 3px solid rgba(59.21568627450981%,19.215686274509817%,10.196078431372547%,0.2);
 
   color: #ffffff;
  font-size: 13px;
  font-family: Helvetica Neue;
  font-weight: 900;
  font-style: normal
  }
QPushButton#pushButton_cmp:pressed {
    background-color: red;
}</string>
    </property>
    <property name="text">
     <string>comparer avec</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_srch">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> 
QPushButton#pushButton_srch {
    
  backface-visibility: hidden;
 
  cursor: pointer;
  
  
  background: #97311a;
  border-radius: 100px;
  border: 3px solid rgba(59.21568627450981%,19.215686274509817%,10.196078431372547%,0.2);
 
   color: #ffffff;
  font-size: 13px;
  font-family: Helvetica Neue;
  font-weight: 900;
  font-style: normal
  }
QPushButton#pushButton_srch:pressed {
    background-color: red;
}</string>
    </property>
    <property name="text">
     <string>Chercher</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Chercher un pays</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Comparer 2 pays</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>751</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
